client/lib/analytics/page-view-tracker/index.jsx

```diff
@@ -68,7 +76,7 @@ export class PageViewTracker extends React.Component {
 		}
 
 		if ( ! delay ) {
-			return recorder( path, title );
+			return recorder( path, title, undefined, properties );
```

Comment:
`undefined` looks super odd here. 

Maybe a sign of a code smell. Since service is optional, we may fold this into the last properties argument. We can leave this for a future refactor though. For now, the two allowed values are `default` which records to ga+tracks, or just `ga`. So let's pass through `'default'` over undefined.

Reply:
Updated on a102071268633b91e8ebd2345867b6d6ad7ecc3f.

Ideally I would like to change the order of the arguments to leave the service last, but haven't checked all the places where `record` is used. Will be janitorialized later.